lib/fs/basicfs_test.go

```diff
@@ -426,12 +423,12 @@ func TestRooted(t *testing.T) {
 			{`\\?\c:\`, `foo`, `\\?\c:\foo`, true},
 			{`c:\`, `\foo`, `c:\foo`, true},
 			{`\\?\c:\`, `\foo`, `\\?\c:\foo`, true},
-			{`c:\`, `\\foo`, ``, false},
-			{`c:\`, ``, ``, false},
-			{`c:\`, `\`, ``, false},
-			{`\\?\c:\`, `\\foo`, ``, false},
-			{`\\?\c:\`, ``, ``, false},
-			{`\\?\c:\`, `\`, ``, false},
+			{`c:\`, `\\foo`, `c:\foo`, true},
```

Comment:
This should not be allowed I think, it should fall under the double Windows pathsep rule (UNC path) and be dropped as it were?